.env

```
@@ -8,4 +8,7 @@ TASK_X_ENV_PRECEDENCE=1
 GOLANGCI_LINT_VERSION=v2.11.3
 
 # renovate: datasource=go depName=k8s.io/code-generator
-DEEPCOPY_GEN_VERSION=v0.35.2
\ No newline at end of file
+DEEPCOPY_GEN_VERSION=v0.35.2
+
+# renovate: datasource=npm depName=markdownlint-cli2
+MARKDOWNLINT_CLI2_VERSION=0.22.0
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)